src/parquet/file/writer-internal.h

```diff
@@ -67,12 +67,14 @@ class SerializedPageWriter : public PageWriter {
 class RowGroupSerializer : public RowGroupWriter::Contents {
  public:
   RowGroupSerializer(int64_t num_rows, const SchemaDescriptor* schema, OutputStream* sink,
-      format::RowGroup* metadata, MemoryAllocator* allocator)
+      format::RowGroup* metadata, MemoryAllocator* allocator,
```

Comment:
Feel free to remove the duplicate allocator argument in this patch. I think it is cascaded right from `ParquetFileWriter::Open()`